extensions/amp-app-banner/0.1/test/test-amp-app-banner.js

```diff
@@ -309,7 +309,7 @@ describe('amp-app-banner', () => {
         meta: {content:
             'app-id=828256236, app-argument=javascript:alert("foo");'},
       }).should.eventually.be.rejectedWith(
-         /The url in app-argument is invalid/);
+         /The url in app-argument has invalid protocol/);
```

Comment:
Please add a test here for no app-argument case ( #9361 ). amp-app-banner should work normally in that case